docker-compose.yml

```diff
@@ -15,23 +15,31 @@ services:
     depends_on:
       - postgres
 
+  # Create EE container
+  conjur-dap:
+    image: registry2.itci.conjur.net/conjur-appliance-cuke-master:5.0-stable
+    ports:
+      - "443:443"
+    security_opt:
+      - "seccomp:unconfined"
+
   conjur-proxy-nginx:
     build:
       context: .
       dockerfile: Dockerfile.nginx
     entrypoint: nginx-debug -g 'daemon off;'
     environment:
-        TERM: xterm
+      TERM: xterm
     ports:
-    - "8080:80"
-    - "8443:443"
+      - "8080:80"
+      - "8443:443"
     depends_on:
       - conjur
       - client
 
 
   client:
```

Comment:
if you leave the environment variables blank in this container spec for the `client` then you can use it for DAP or Conjur OSS - in the script before the container is created, you export the values that you need before you run `docker-compose up -d client`

this would mean you could run
```
export CONJUR_APPLIANCE_URL="http://conjur:3000"
docker-compose up -d client
docker-compose exec client ... # (run commands on the client connected to the OSS instance)
docker-compose down client
...
# later in the file
export CONJUR_APPLIANCE_URL="http://conjur-dap:443"
docker-compose up -d client
docker-compose exec client ... # (run commands on the client connected to the DAP instance)
```